### ISSUE TITLE
db:migrate Database error code: 25001 PreventInTransactionBloc

### ISSUE THREAD

**Issue by issambox** [baptisteArno/typebot.io#934]
On PostgreSQL 15.4 i have this issue when execute migration with pnpm db:migrate

:~/typebot.io$ pnpm db:migrate

> typebot-os@2.18.0 db:migrate /home/typebot/typebot.io
> cd packages/prisma && pnpm run db:migrate

> @typebot.io/prisma@0.1.0 db:migrate /home/typebot/typebot.io/packages/prisma
> pnpm migrate:deploy

> @typebot.io/prisma@0.1.0 migrate:deploy /home/typebot/typebot.io/packages/prisma
> dotenv -e ./.env -e ../../.env -- tsx scripts/migrate-deploy.ts

Executing for PostgreSQL schema
Command failed: prisma migrate deploy --schema postgresql/schema.prisma
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20220513223344_add_workspaces

Database error code: 25001

Database error:
ERREUR: ALTER TYPE ... ADD ne peut pas être exécuté dans un bloc de transaction

DbError { severity: "ERREUR", parsed_severity: Some(Error), code: SqlState(E25001), message: "ALTER TYPE ... ADD ne peut pas être exécuté dans un bloc de transaction", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("xact.c"), line: Some(3237), routine: Some("PreventInTransactionBlock") }

**Comment by issambox** [comment]
I've encountered this error following a manual installation of Typebot. I have set up a PostgreSQL server to host the project's database. 
When I execute the following command for database migration: "pnpm prisma generate && pnpm db:migrate," I encounter the following error:

**Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20220513223344_add_workspaces

Database error code: 25001

Database error:
ERROR: ALTER TYPE ... ADD cannot be executed within a transaction block.

DbError { severity: "ERREUR", parsed_severity: Some(Error), code: SqlState(E25001), message: "ERROR: ALTER TYPE ... ADD cannot be executed within a transaction block.", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("xact.c"), line: Some(3237), routine: Some("PreventInTransactionBlock") }**


For your information, I conducted a test on both PostgreSQL 12 and 15. The application runs, but I encounter an error in the browser stating "404 bot not found." I believe this issue might be related to the absence of the database.
